src/main/resources/templates/fragments/head.html

```diff
@@ -6,5 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="format-detection" content="telephone=no">
   <title th:text="${title}"></title>
+  <th:block
```

Comment:
Actually I don't think it's a good idea to put [rendered content inside the `<head>`](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML)

> The head of an HTML document is the part that is _not displayed_ in the web browser when the page is loaded.

But your idea to put it in the `head` fragment is still a good one.

My thought it we could do something like this:

```html
<th:block th:fragment="head" th:assert="${!#strings.isEmpty(title)}">
  <head>...</head>
  <th:block th:replace="demoBannter" />
</th:block>
```

We can pair on this if you'd like. Or welcome to pushback.

Reply:
Ugggghhh actually it's more complicated, as all valid HTML that [visually appears on the screen needs to be within the `<body>` tag](https://teamtreehouse.com/community/why-is-it-not-valid-to-place-elements-outside-when-body-element-it-is-used-as-the-wrapper).

Reply:
Ok, next best solution: we add `demoBanner` into the `toolbar` fragment (either above or below). I assume every screen/static page will use the `toolbar` as that's where our logo is.